edifact/models/tests/segments_comparison_util.py

```diff
@@ -0,0 +1,49 @@
+import unittest
+
+from edifact.models.interchange import InterchangeHeader, InterchangeTrailer
+from edifact.models.message import MessageHeader, MessageTrailer, NameAndAddress, DateTimePeriod, \
+    ReferenceTransactionType, ReferenceTransactionNumber
+
+
+class SegmentComparisonTest(unittest.TestCase):
```

Comment:
your file name does not match your class name?

Reply:
yes, it doesn't -> thank you for spotting this :)